src/vivarium_cluster_tools/psimulate/worker/vivarium_work_horse.py

```diff
@@ -97,15 +97,11 @@ def work_horse(job_parameters: dict) -> Tuple[pd.DataFrame, Dict[str, pd.DataFra
 
         results = sim.get_results()  # Dict[measure, results dataframe]
 
-        idx = pd.MultiIndex.from_tuples(
-            [(job_parameters.input_draw, job_parameters.random_seed)],
-            names=["input_draw_number", "random_seed"],
-        )
-        finished_results_metadata = pd.DataFrame(index=idx)
+        finished_results_metadata = pd.DataFrame(index=[0])
```

Comment:
Is the `index` argument necessary?

Reply:
It is! That's the bug I introduced yesterday. Without it, no actual values get added

```
>>> import pandas as pd
>>> df1 = pd.DataFrame()
>>> df1["foo"] = "bar"
>>> df1
Empty DataFrame
Columns: [foo]
Index: []
>>> df2 = pd.DataFrame(index=[0])
>>> df2["foo"] = "bar"
>>> df2
   foo
0  bar
>>> 
```